ui/app/utils/clickhouse/common.ts

```diff
@@ -232,8 +232,8 @@ export const requestMessageSchema = z.object({
 export type RequestMessage = z.infer<typeof requestMessageSchema>;
 
 export const jsonInferenceOutputSchema = z.object({
-  raw: z.string(),
-  parsed: z.any().optional(),
+  raw: z.string().nullable(),
```

Comment:
I think what you need to do here is have it default to "" instead of making this optional everwhere in the codebase. We can keep the null encoding for primary storage but for now I don't think we're gonna handle this case in the UI so I would prefer to keep the code simple.